test/documentSymbols.test.ts

```diff
@@ -298,6 +298,28 @@ suite('Document Symbols Tests', () => {
             );
         });
 
+        it('Document Symbols with complex mapping and aliases', () => {
+            const content = `version: 0.0.1
+            structure:
+              ? &root root
+              :
+                element: div
+            conditions:
+              ? *root
+              :
+                style:
+                  height: 41
+            `;
+
+            try {
```

Comment:
Do we need the try catch here? wont `const symbols = parseHierarchicalSetup(content);` just error the test which is probably more what we want